azure-devops-extension/setup.py

```diff
@@ -54,9 +54,9 @@
     license='MIT',
     description="Azure DevOps Command Line Interface",
     long_description=README + '\n\n' + HISTORY,
-    author="Microsoft Corporation",
-    author_email="vstscli@microsoft.com",
-    url="https://github.com/Microsoft/vsts-cli",
+    author="Microsoft",
+    author_email="VSTS_Social@microsoft.com",
+    url="https://github.com/Microsoft/vsts-cli/tree/azuredevopscli-dev",
```

Comment:
Why to change url? we plan to make this default branch anyways and then the repo url will point to the new code.

Reply:
till we make this default branch